src/http/server/handlers/static_file_handler.cr

```diff
@@ -49,16 +53,17 @@ class HTTP::StaticFileHandler
 
     file_path = File.join(@public_dir, expanded_path)
     is_dir = Dir.exists? file_path
+    is_file = File.exists?(file_path) && !is_dir
```

Comment:
Minor improvement: changing the order means one less system call for dirs: `!is_dir && File.exists?(file_path)`